src/Exception/UserException.php

```diff
@@ -4,14 +4,34 @@
 
 namespace Yiisoft\ErrorHandler\Exception;
 
+use Attribute;
 use Exception;
+use ReflectionClass;
+use Throwable;
+
+use function count;
 
 /**
- * UserException is the base class for exceptions that are meant to be shown to end users.
- * Such exceptions are often caused by mistakes of end users.
+ * `UserException` is an exception anв class attribute that indicates
```

Comment:
```suggestion
 * `UserException` is an exception and class attribute that indicates
```

Please check all the text to catch typos